mapsforge-map/src/main/java/org/mapsforge/map/layer/Layer.java

```diff
@@ -178,7 +182,7 @@ final synchronized void assign(Redrawer redrawer) {
 
     final synchronized void unassign() {
         if (this.assignedRedrawer == null) {
-            throw new IllegalStateException("layer is not assigned");
+            LOGGER.warning("layer is not assigned");
```

Comment:
Same here, probably it is better to `return;` after the warning?

Reply:
similar to above